packages/fxa-settings/src/pages/ResetPassword/index.tsx

```diff
@@ -194,6 +195,8 @@ const ResetPassword = ({
       </form>
 
       <LinkRememberPassword {...{ email }} />
+
+      <ThirdPartyAuth {...{ enabled: true }}></ThirdPartyAuth>
```

Comment:
Q: Should this be self-closing (re: `packages/fxa-settings/src/pages/Signin/index.tsx#L136` below)?
```html
<ThirdPartyAuth {...{ enabled: true }} />
```

Reply:
This should be false woops